javascript/javascript3/week1/Giphy_api.js

```diff
@@ -0,0 +1,42 @@
+
+let button = document.getElementById('mybutton');
+button.addEventListener('click', fetchGif);
+
+function fetchGif() {
+    let gifType = document.getElementById('giftype');
+    let gif = gifType.value;
+    let noOfGifs = document.getElementById('gifnum');
+    let Number = noOfGifs.value;
```

Comment:
Why is `Number` capital? Also be aware that `Number` with a capital `N` is reserved in the Javascript namespace, because it is a Datatype. One should not used the words which are reserved in Javascript. e.g. never used `let` as a variable name :)